### PR TITLE
🔥 Remove `melos_` prefix for IDEA modules

### PR DESCRIPTION
**Pull request by AlexV525**
### New Pull Request Checklist

- [x] I have read the [Documentation](https://pub.dev/documentation/dio/latest/)
- [x] I have searched for a similar pull request in the [project](https://github.com/cfug/dio/pulls) and found none
- [x] I have updated this branch with the latest `main` branch to avoid conflicts (via merge from master or rebase)
- [ ] I have added the required tests to prove the fix/feature I'm adding
- [ ] I have updated the documentation (if necessary)
- [x] I have run the tests without failures
- [ ] I have updated the `CHANGELOG.md` in the corresponding package

### CODE REVIEW

File: melos.yaml [hunk 1]
```diff
@@ -7,4 +7,6 @@ packages:
   - example*
 
 ide:
-  intellij: true
+  intellij:
+    enabled: true
+    moduleNamePrefix: ''
```
Comment (AlexV525) [inline melos.yaml:12]: Core

**Review by ueman — APPROVED** [review]
I’m not a IntelliJ user, so this gets a RSLGTM

**Comment by AlexV525** [comment]
> I’m not a IntelliJ user, so this gets a RSLGTM

Haha, @kuhnroyal would love this

**Comment by kuhnroyal** [comment]
Gonna check this when I am back from vacation. Didn't know about the prefix.

**Pull request merged**